src/App.tsx

```diff
@@ -1,7 +1,39 @@
-import React from 'react';
+import { Component } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+type State = {
+  keyPress: string,
+};
+
+export class App extends Component<{}, State> {
+  state = {
+    keyPress: '',
+  };
+
+  componentDidMount() {
+    document.addEventListener('keyup', this.documentCLick);
+  }
+
+  componentWillUnmount() {
+    document.removeEventListener('keyup', this.documentCLick);
+  }
+
+  documentCLick = (event: KeyboardEvent) => {
+    const keyPress = event.key;
+
+    this.setState({ keyPress });
+  };
+
+  render() {
+    const { keyPress } = this.state;
+
+    return (
+      <div className="App">
+        <p className="App__keyPress">
```

Comment:
```suggestion
        <p className="App__message">
```
I think this is the reason why your tests failed. Please fix it
![image](https://user-images.githubusercontent.com/96409563/197127905-4283bd41-ea34-4ece-8eb5-3baab4e9f1b5.png)